Dockerfile

```diff
@@ -35,7 +35,7 @@ COPY . .
 COPY --from=ui-builder /ui/public /app/web/public
 
 #RUN CGO_ENABLED=0 GOOS=linux go build -o health cmd/health/main.go
-RUN CGO_ENABLED=0 GOOS=linux go build -o keploy cmd/server/main.go
+RUN CGO_ENABLED=0 GOOS=linux go build -o keploy cmd/keploy-cli/main.go
```

Comment:
any reason to change the package names? servers can have command line flags.

Reply:
Coz I didn't like the package name `"go.keploy.io/server/cmd/server/cmd"`. But since you insist, I have made the change.

Reply:
I agree I dont like that package name either.

Reply:
how about we just move the contents of the `go.keploy.io/server/cmd/server/cmd/root.go` to `go.keploy.io/server/cmd/server/main.go`? I know it makes the current main.go less "clean", but we can move it to a dedicated folder once we have a sufficiently large number of CLI commands.

Reply:
No, because that breaks the standard command pattern for Go CLIs. We also have #206 to add another command and sooner or later we will have more.

Reply:
@gauravgahlot can you please explain why would it break the standard command pattern? 

I see this as a common practice to include CLI-related code in many go server projects. eg: 

https://github.com/temporalio/temporal/blob/master/cmd/server/main.go
https://github.com/minio/minio/blob/master/cmd/main.go
https://github.com/caddyserver/caddy/blob/master/cmd/main.go

Just to clarify, these projects could definitely be using older design patterns because the best practices evolve. But would love to understand and have a healthy discussion.

Reply:
I prefer to do things the right way, if know the "right way". Like in this case, there is a good probability that the CLI will have more commands added going forward. So, I would prefer to do things the way they should be - clean code.

Reply:
It hardly matters whether you put the code block in main.go or root.go. What matter the most, at least for me, are the coding standards and design principles I have learned in my small journey so far.

Reply:
I really dont like creating `cmd/server/cmd` package. If we just put `root.go` in `cmd/server` then we'll have to change build commands everywhere from `go run cmd/server/main.go` to something like `go run cmd/server/*.go` which adds additional steps in setting up IDEs and requires changing all build-scripts/Dockerfiles. 

So I see 3 possible ways: 

1. Add a separate top-level CLI package,  so something like `cli/cli.go`. I feel this is overkill just for a few packages and would hamper readability. 
2. Move the content to something like `server/cli.go`. This can work because the CLIs are currently written for the server, but need to be refactored once we add non-server-related CLI commands. 
3. Just put it into `main.go`. Would need refactoring once there are 5+ CLI commands but its less ambiguity and easier to read.

Reply:
There's no single "right way" to do anything, it's always relative and can change according to the situation. What works for a 10k lines project might differ from a 1000 or 100k lines project and vice versa. These are subjective and we should learn to reason with our practices and adapt.